specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-10-02-preview/managedClusters.json

```diff
@@ -4088,10 +4100,221 @@
           },
           "x-ms-identifiers": [],
           "description": "Time slots on which upgrade is not allowed."
+        },
+        "maintenanceWindow": {
+          "type": "object",
+          "$ref": "#/definitions/MaintenanceWindow",
+          "description": "Maintenance window for the maintenance configuration."
         }
       },
       "description": "Properties used to configure planned maintenance for a Managed Cluster."
     },
+    "MaintenanceWindow": {
+      "type": "object",
+      "properties": {
+        "schedule": {
+          "$ref": "#/definitions/Schedule",
+          "description": "Recurrence schedule for the maintenance window."
+        },
+        "durationHours": {
+          "type": "integer",
+          "format": "int32",
+          "minimum": 4,
+          "maximum": 24,
+          "default": 24,
+          "description": "Length of maintenance window range from 4 to 24 hours."
+        },
+        "utcOffset": {
+          "type": "string",
+          "pattern": "^(-|\\+)[0-9]{2}:[0-9]{2}$",
+          "description": "The UTC offset in format +/-HH:mm. For example, '+05:30' for IST and '-07:00' for PST. If not specified, the default is '+00:00'."
+        },
+        "startDate": {
+          "type": "string",
+          "format": "date",
+          "description": "The date the maintenance window activates. If the current date is before this date, the maintenance window is inactive and will not be used for upgrades. If not specified, the maintenance window will be active right away."
+        },
+        "startTime": {
+          "type": "string",
+          "pattern": "^\\d{2}:\\d{2}$",
+          "description": "The start time of the maintenance window. Accepted values are from '00:00' to '23:59'. 'utcOffset' applies to this field. For example: '02:00' with 'utcOffset: +02:00' means UTC time '00:00'."
+        },
+        "notAllowedDates": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DateSpan"
+          },
+          "x-ms-identifiers": [],
```

Comment:
I don't think you want/need this `x-ms-identifiers`

Reply:
Fixed, thank you!!

Reply:
Well, I thought this is kind of necessary to pass the check in [Swagger LintDiff](https://github.com/Azure/azure-rest-api-specs/pull/21337/checks?check_run_id=9231250516).

Since this is an array, the spec will expect each element to be distinguishable, but an empty list can also be used to bypass the check.

Reply:
Huh, interesting. Looks like you need to bring this back then. TIL.

Sorry for the bad advice @wenxuan0923.

Reply:
Hi @matthchr and @FumingZhang , thank you both for your comments! I have added the identifiers back.